lib/builder.js

```diff
@@ -93,6 +93,35 @@ class MessageBuilder {
         return this;
     }
 
+    /**
+     * Set the embedded url
+     * @param {string} url The url 
+     */
+    setURL(url) {
+        this.data.attachments[0].url = url;
+        return this;
+    }
+
+    /**
+     * Set the author of the embedded object
+     * @param {string} author the author
+     * @param {string} iconURL Optional: the icon url
+     * @param {string} url Optional: the url
+     */
+    setAuthor(author, iconURL, url) {
+        this.data.attachments[0].author = author;
```

Comment:
Wrong key name, should be `author_name`:
```suggestion
        this.data.attachments[0].author_name = author;
```